lib/xcode.js

```diff
@@ -229,6 +229,31 @@ const getMaxIOSSDK = _.memoize(
   }
 );
 
+async function getMaxTVOSSDKWithoutRetry (timeout = XCRUN_TIMEOUT) {
+  const version = await getVersion(false, DEFAULT_NUMBER_OF_RETRIES, timeout);
+  if (version[0] === '4') {
```

Comment:
As far as I can judge, this is basically forcing treat xcode 4 to be treated as xcode 6.1.
It was implemented way back here: https://github.com/appium/appium-xcode/pull/2/files.

I honestly don't know why is it implemented that way, but it feels like I shouldn't ignore it 😄.